src/createListComponent.js

```diff
@@ -441,13 +442,13 @@ export default function createListComponent({
             if (!sizeOfNextElement && this.state.scrolledToInitIndex) {
               this.setState(prevState => {
                 if (
-                  prevState.localOlderPostsToRender &&
-                  prevState.localOlderPostsToRender[0] !== overscanStopIndex + 1
+                  prevState.localOlderPostsToRender[0] !==
+                  overscanStopIndex + 1
```

Comment:
Initially i had null so had this check to prevent issues. 
localOlderPostsToRender has an empty array initialised in constructor now. So this is not needed. 
Though it doesn't hurt to leave i don't see a case where we will have a value which would cause this to fail